### ISSUE TITLE
Feature Request - Limit Number of Articles Displayed on Document index page

### ISSUE THREAD

**Issue by ese7** [weDevsOfficial/wedocs-plugin#46]
Hi @tareq1988 

Can there be a setting or shortcode parameter that allows you specify the maximum number of articles to display per doc? 

For instance, if I have Doc A (with 5 articles) and Doc B (with 7 articles); and I specify say 3 as maximum number of articles, it displays not more than 3 articles each under Doc A and Doc B on the index page, then there can be a **View all** button to expand the list if there are more articles.

I hope this clarifies :-)

**Comment by Rubaiyat-E-Mohammad** [comment]
I do not think this feature will add any significant value to the plugin. Please have a look. @anik-fahmid vai
